server/views/pages/order/about-the-device-wearer/responsible-adult-details.njk

```diff
@@ -38,7 +38,7 @@
     }) }}
   </div>
 
-  <form method="post">
+  <form action="{{ formActionUri }}" method="post">
```

Comment:
We shouldn't need `formActionUri` because we should just be submitting to the current page